packages/devextreme/js/__internal/grids/new/card_view/content_view/content/card/header.tsx

```diff
@@ -80,24 +80,50 @@ export class CardHeader extends Component<CardHeaderProps> {
 
     const checkBoxItem = this.getCheckBoxItem();
 
-    const captionItem: dxToolbar.Item | null = captionExpr && card?.[captionExpr]
-      ? { location: 'before', text: card[captionExpr] }
-      : null;
+    const captionItem = !!captionExpr && card?.[captionExpr] && {
+      name: 'caption',
+      location: 'before',
+      text: card[captionExpr],
+    } satisfies dxToolbar.Item & { name: string };
+
+    const updateButton = allowUpdating && {
+      name: 'updateButton',
+      location: 'after',
+      widget: 'dxButton',
+      options: { icon: 'edit', onClick: onEdit, stylingMode: 'text' },
+    } satisfies dxToolbar.Item & { name: string };
+
+    const deleteButton = allowDeleting && {
+      name: 'deleteButton',
+      location: 'after',
+      widget: 'dxButton',
+      options: { icon: 'remove', onClick: onDelete, stylingMode: 'text' },
```

Comment:
Icon should be ```trash``` here
```suggestion
      options: { icon: 'trash', onClick: onDelete, stylingMode: 'text' },
```

Reply:
fixed